src/editor/pages/settings.tsx

```diff
@@ -0,0 +1,82 @@
+import { NextPage } from 'next'
+import Head from 'next/head'
+import { useState } from 'react'
+import { List } from '../components/list/list'
+import { PageTitle } from '../components/page-title'
+import { TokenListItem } from '../components/settings/token-list-item'
+import { TokenInsertNew } from '../components/settings/token-insert-new'
+import { Message, MessageType } from '../components/message'
+
+interface Token {
+  Id: string;
+  Name: string;
+}
+
+interface SettingsProps {
+  tokens: Token[];
+}
+
+export const Settings: NextPage<SettingsProps> = ( { tokens: t }: SettingsProps ) => {
+  const [tokens, setTokens] = useState( t )
+  const [globalMessage, setMessage] = useState<Message>( )
+
+  const updateMessage =( message: Message ): void => {
+    setMessage( message )
+    setTimeout( () => setMessage( undefined ), 1500 )
+  }
+
+  const deleteToken = async( id: string ): Promise<void> => {
+    const res = await fetch( `/api/token/delete`, { method: 'DELETE', body: JSON.stringify( { id } ) } )
```

Comment:
use query params instead of body. Usually a request with the DELETE method isn't expected to have a body.